iphone/Maps/Classes/Widgets/MWMMapDownloadDialog.mm

```diff
@@ -219,6 +221,22 @@ - (void)processViewportCountryEvent:(CountryId const &)countryId {
     [self configDialog];
 }
 
+// Create a font with the new descriptor and specified size
+- (UIFont *)monospacedFontWithSize:(CGFloat)size {
```

Comment:
@biodranik i think that in the case of a font, it will be useful to create an extension rather than use a local fix. 
```swift
extension UIFont {
  @objc
  var monospaced: UIFont {
...
```

So it will be reusable like:
`self.nodeSize.font = UIfont.medium14.monospaced;`

WDYT?

Reply:
Created an extension according to suggestion.